report/2.Analysis/MotionTracking.tex

```diff
@@ -0,0 +1,46 @@
+\section{Object Tracking}
+
+To be able to track or know where the target is located, the device will require some sensors that can transmit information about the position of the desired target.
+One could easily imagine a couple of different sensors that could potentially gather information about the position of the target.
+For instance laser and sonar could both be used to gather some information about the position of a target\cite{Sensors}.
+However, since the system should be adaptable to a dynamic environment with potentially a lot of different objects in motion, one sensor was deemed the most obvious choice for this system; a camera.
+
+Using a camera is powerful, since images can be processed in order to determine what objects are present in the image.
```

Comment:
Using a camera is powerful, Lyder mærkeligt for mig, kan dog godt være det faktisk er i orden